src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -36,29 +36,35 @@
 import org.zaproxy.zap.extension.pscan.PassiveScanThread;
 import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
 
+/**
+ * Plugin that scans the content for private IP V4 addresses as well as 
+ * Amazon EC2 private hostnames (for example, ip-12-34-56-78).
+ */
 public class TestInfoPrivateAddressDisclosure extends PluginPassiveScanner {
 
-	/**
-	 * Prefix for internationalised messages used by this rule
-	 */
-	private static final String MESSAGE_PREFIX = "pscanrules.testinfoprivateaddressdisclosure.";
+    /**
+     * Prefix for internationalised messages used by this rule
+     */
+    private static final String MESSAGE_PREFIX = "pscanrules.testinfoprivateaddressdisclosure.";
 	
     private static final String REGULAR_IP_OCTET = "\\b(25[0-5]|2[0-4][0-9]|1?[0-9]{1,2})";
-    private static final String REGULAR_PORTS = "\\b(6553[0-5]|65[0-5][0-2][0-9]|6[0-4][0-9]{4}|[0-5]?[0-9]{0,4})";
+    private static final String REGULAR_PORTS = "\\b(6553[0-5]|65[0-5][0-2][0-9]|6[0-4][0-9]{3}|[0-5]?[0-9]{0,4})";

```

Comment:
@thc202 @schoeth I think I've identified another mistake in this RegEx. I believe it should also end with an word boundry marker `\\b` otherwise `:75000` matches on the first 4 characters (_7500_).

Original: https://regex101.com/r/uO1rI2/2
Proposed fix: https://regex101.com/r/uO1rI2/3
Note: Regex101 syntax is slightly different, due to escaping requirements.